Code/Engine/JsonManager.cpp

```diff
@@ -0,0 +1,74 @@
+#include "sadpch.h"
+
+#include "JsonManager.h"
+
+using json = nlohmann::json;
+//Constructor
+sad::JsonManager::JsonManager(){}
+
+/**
+ * @brief 
+ * return the entire json data
+ * @sample
+ * std::string jsonPath = core::FileUtils::GetDataDirectory() + core::FileUtils::ConvertOSPathString("/TestFolder/sample.json");
+ * @param path 
+ * @return 
+*/
+json sad::JsonManager::GetJson(std::string path)
+{
+	std::string fileContent = core::FileUtils::ReadFile(path);
+	return json::parse(fileContent);
+}
+
+/**
+ * @brief 
+ * Adds entities of the last saved level
+ * Level 0 is the default location
+ * saveFile default value is 1
+*/
+bool sad::JsonManager::ImportLevel(int saveFile)
+{
+	//if the savefile location is out of range
+	if (0 > saveFile && saveFile < 100)
+		return false;
+
+	std::stringstream ss;
+	ss << std::setw(2) << std::setfill('0') << saveFile;
+	std::string level_path = core::FileUtils::GetPathInsideDataDirectory("/Save/SaveFile" + ss.str() + ".json");
+
+	if (!core::FileUtils::PathExists(level_path))
+		return false;
```

Comment:
I know we don't have a 'default level' yet, but can we add a log with `ELogType::Warn` or `ELogType::Error` describing this case?